apps/site/pages/en/index.mdx

```diff
@@ -17,15 +17,36 @@ layout: home
   <div className="flex gap-4">
 
     <div className="flex flex-col gap-2">
-      <Button kind="special" className="!hidden dark:!block" href="/download">Install Node.js</Button>
+      <Button kind="special" className="!hidden dark:!block" href="/download">Get Node.j®</Button>
 
-      <Button kind="primary" className="!block dark:!hidden" href="/download">Install Node.js</Button>
+      <Button kind="primary" className="!block dark:!hidden" href="/download">Get Node.js®</Button>
 
       <Button kind="secondary" className="!block" href="/blog/announcements/node-18-eol-support">
         <span>Get security support</span>
         <br />
         <small className="!text-xs">for Node.js 18 and below</small>
       </Button>
+
+      <div className="flex flex-row gap-2 justify-center xs:mt-6">
+        <WithNodeRelease status="Current">
+          {({ release }) =>
+            <BadgeGroup size="small" badgeText={release.versionWithPrefix} href={`/blog/release/${release.versionWithPrefix}`}>
+              Node.js Current
+            </BadgeGroup>}
+        </WithNodeRelease>
+
+        <WithNodeRelease status="LTS">
+          {({ release }) =>
+            <BadgeGroup size="small" kind="warning" badgeText={release.versionWithPrefix} href={`/blog/release/${release.versionWithPrefix}`}>
+              Node.js LTS
+            </BadgeGroup>}
+        </WithNodeRelease>
+      </div>
+
+      <span className="italic text-center text-xs! text-gray-500">
+        <span>"LTS" stands for the long-term support version of Node.js. </span>
+        <span>Learn more about Node.js releases <Link href="/about/previous-releases">here</Link>.</span>
```

Comment:
It's not very accessible for the link content to be "here", rather, the link content should be "Learn more about Node.js releases"